### DISCUSSION TITLE
Setting `defaultCommentOrder` to `newest` but it doesn't seem to update

### DISCUSSION THREAD

**Discussion by LukeTwomey** [giscus/giscus#878] [Q&A] [answered]
I've added `defaultCommentOrder="newest"` to the Giscus config, but when it renders on the page, the default still seems to be "Oldest":

<img width="278" alt="image" src="https://user-images.githubusercontent.com/7040126/216098691-6b6d07d4-16aa-4357-9b40-542e40d59968.png">

The default "Oldest" button is still applied when the [page](https://luketheweb.dev/blog/how-to-become-a-web-developer) loads:

<img width="695" alt="image" src="https://user-images.githubusercontent.com/7040126/216099845-cbc98bfc-b956-449f-a002-1a2aaf7e0b80.png">

**Comment by laymonage** [comment, marked as answer]
This setting is currently only supported through [`giscus.json`](https://github.com/giscus/giscus/blob/main/ADVANCED-USAGE.md#defaultcommentorder), not a data attribute.

**Reply by LukeTwomey** [reply]
Ah thank you, missed that.
